applications_tests/lethe-fluid/heat_transfer_vof_laser_material_interface.output

```diff
@@ -0,0 +1,64 @@
+Running on 1 MPI rank(s)...
+   Number of active cells:       625
+   Number of degrees of freedom: 2028
+   Volume of triangulation:      0.04
+   Number of thermal degrees of freedom: 676
+   Number of VOF degrees of freedom: 676
+
+*******************************************************************************
+Transient iteration: 1        Time: 0.005    Time step: 0.005    CFL: 0       
+*******************************************************************************
+----
+VOF
+----
+Newton iteration: 0  - Residual:  0
+  -Tolerance of iterative solver is : 1e-08
+  -Iterative solver took : 0 steps 
+		alpha =      1 res = 0
+---------------
+Fluid Dynamics
+---------------
+Newton iteration: 0  - Residual:  0
+  -Tolerance of iterative solver is : 1e-08
+  -Iterative solver took : 0 steps 
+		alpha =      1 res = 0
+--------------
+Heat Transfer
+--------------
+Newton iteration: 0  - Residual:  1132
+  -Tolerance of iterative solver is : 1.132
+  -Iterative solver took : 11 steps 
+		alpha =      1 res = 0.4807
+Newton iteration: 1  - Residual:  0.4807
+  -Tolerance of iterative solver is : 0.0004807
+  -Iterative solver took : 13 steps 
+		alpha =      1 res = 0.0002065
```

Comment:
don't put the tolerances and the verbosity of the non-linear and linear solver within tests output,  those are rarely portable across machines